Results_Engine/Results_Engine.csproj

```diff
@@ -66,6 +81,40 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Analytical_Engine\Analytical_Engine.csproj">
+      <Project>{b3342b93-fdbc-465b-b2e7-da463589c96b}</Project>
+      <Name>Analytical_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\BHoM_Engine\BHoM_Engine.csproj">
+      <Project>{1ad45c88-dd54-48e5-951f-55edfeb70e35}</Project>
+      <Name>BHoM_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Geometry_Engine\Geometry_Engine.csproj">
+      <Project>{89ab2dcb-ef87-4cba-b59c-c16a8a71d333}</Project>
+      <Name>Geometry_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Graphics_Engine\Graphics_Engine.csproj">
+      <Project>{d5c7704d-f59a-4fc6-8d1e-356699a174e0}</Project>
+      <Name>Graphics_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Library_Engine\Library_Engine.csproj">
+      <Project>{fb4ed5c8-e7cc-44b3-95eb-95c304d54e5d}</Project>
+      <Name>Library_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Reflection_Engine\Reflection_Engine.csproj">
+      <Project>{b0154405-9390-472d-9b5c-a2280823b18d}</Project>
+      <Name>Reflection_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Spatial_Engine\Spatial_Engine.csproj">
+      <Project>{a84fdfe5-c267-430e-8fec-af4f4bf5b745}</Project>
+      <Name>Spatial_Engine</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\Structure_Engine\Structure_Engine.csproj">
```

Comment:
Is this reference required, or leftover from prototyping and testing? Would be good if we could get away with not having this reference in.

Reply:
I think we should be able to get rid of some references, Structure_oM is required due to MeshResultSmoothingType but I don't think Structure_Engine should be. I don't personally have time to remove and test today unfortunately, so unless someone else does it earlier I'll clean up the references next week.

Reply:
Yeah, keeping Structure_oM is fine, it was already in here, required by the envelope methods for BarForces (might need to move that out at one point as well, but that is for sure nothing that this PR should have to worry about!)
Was mainly concerned of any (hopefully) unnecessary engine cross dependencies, as I can see a case of the structures_Engine wanting to depend on results engine going forward.

Reply:
Also, no rush to get this done today! I will struggle for time to fully review today anyway, just spotted it and put a comment when I saw it!

Reply:
The Structure_Engine references and a couple other unused ones added by this have been removed.